include/substrait-mlir/Dialect/Substrait/IR/SubstraitOps.td

```diff
@@ -526,8 +526,10 @@ def Substrait_ProjectOp : Substrait_RelOp<"project", [
   }];
 }
 
-def SetOpKind : I32EnumAttr<"SetOpKind", [{The enum values correspond to those 
-in the SetRel.SetOp message.}],[I32EnumAttrCase<"unspecified", 0>,
+def SetOpKind : I32EnumAttr<"SetOpKind",
+  "The enum values correspond to those in the SetRel.SetOp message.", [
+    I32EnumAttrCase<"unspecified", 0>,
+  // Remaining cases indented by four spaces
```

Comment:
Haha, sorry, that wasn't clear enough. I meant that to be an instruction to you: please indent the remaining cases by four spaces rather than two such that they are aligned with the `unspecified` case.

Reply:
Oopsie sorry about that 😆